applications/pionix_chargebridge/include/charge_bridge/can_bridge.hpp

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2025 Pionix GmbH and Contributors to EVerest
+#pragma once
+
+#include <chrono>
+#include <everest/io/can/can_payload.hpp>
+#include <everest/io/can/socket_can.hpp>
+#include <everest/io/event/fd_event_register_interface.hpp>
+#include <everest/io/event/timer_fd.hpp>
+#include <everest/io/udp/udp_client.hpp>
+#include <memory>
+
+extern "C" struct cb_can_message;
+
+namespace charge_bridge {
+
+struct can_bridge_config {
```

Comment:
structs and classes in EVerest typically use CamelCase for their name and snake_case for members